bokehjs/src/coffee/renderer/annotation/span.coffee

```diff
@@ -14,13 +13,21 @@ class SpanView extends PlotWidget
     @$el.hide()
 
   bind_bokeh_events: () ->
-    @listenTo(@model, 'change:location', @_draw_span)
+    if @mget('for_hover')
```

Comment:
My intent was really that this was a initialization configuration, maybe there is a better way to express that than making it a BB property.